### ISSUE TITLE
API Platform v3 support

### ISSUE THREAD

**Issue by viktorandreevichmorozov** [metaclass-nl/filter-bundle#17]
```
composer require metaclass-nl/filter-bundle
...
Your requirements could not be resolved to an installable set of packages.

Problem 1
    - Root composer.json requires metaclass-nl/filter-bundle * -> satisfiable by metaclass-nl/filter-bundle[v1.0.2, v1.0.3, v2.0.1].
    - metaclass-nl/filter-bundle[v1.0.2, ..., v1.0.3, v2.0.1] require api-platform/core ^2.5 -> found api-platform/core[v2.5.0, ..., v2.7.9] but it conflicts with your root composer.json require (^3.0).
...
```

**Comment by metaclass-nl** [comment]
Maybe this happened because v3.0.0.rc2 for /core 3.0 was marked as a pre-release. I just merged branch 3.0 into master and made a final release v3.0.0, maybe it works now?

**Comment by metaclass-nl** [comment]
No response, probably works
